### ISSUE TITLE
Installation Instructions for Mac OSX are broken due to wrong innoextract flag?

### ISSUE THREAD

**Issue by guy4261** [alexbatalov/fallout1-ce#163]
On my machine, I see `innoextract` should be probably used with `-d`, not `-I`:

```shell
> innoextract -v
innoextract 1.9
Extracts installers created by Inno Setup 1.2.10 to 6.0.5

> innoextract --help 2>&1 | grep '\-I'
  -I [ --include ] arg          Extract only files that match this path
 
> innoextract --help 2>&1 | grep 'output-dir'
  -d [ --output-dir ] arg       Extract files into the given directory
```

So I would suggest:

1. Buy the game from GoG. Don't download their (Windows) installer but the two files the link two (`setup_fallout_1.2_(27130).exe` - 800K in size and `setup_fallout_1.2_(27130)-1.bin` - 686M in size from what I've seen).

2.  As suggested in the current instructions, `brew install innoextract`. For me it produced version 1.9 as seen above.

3. `innoextract setup_fallout_1.2_\(27130\).exe -d /Applications/Fallout`

4. Download the `fallout-ce-macos.dmg`, open it, pull the `Fallout Community Edition` app icon into `/Applications/Fallout`.

5. Running the app will now successfully start the game.

**Comment by weberc2** [comment]
It would also be helpful to instruct people to download the game using the offline installers. The main GOG download link gives you `GOG_Galaxy_Fallout.exe` which I have no idea what to do with. I had to dig around to find the offline installer buried under several menus.

**Comment by guy4261** [comment]
I'm a mac user so I can't really run the offline installer. My experience (and I bet these screens change often...) is that the default option is to download an offline installer, but there's also a list with two items that lets you download the 2 files I mentioned.

**Comment by aalchalabi** [comment]
Following these instructions did succesfully get me the files for the game. However the community Fallout app doesn't start. It just keeps bouncing and says Application not responding after a while. I'm using Sonoma 14.4.1. Other than that I have no weird stuff installed. Anyone know what could be wrong?

**Comment by weberc2** [comment]
In addition to issues with the `innoextract` flag, the official instructions don't make sense for the steam download which doesn't give you an installer but rather it gives you a couple `.DAT` files and a `.cfg` file. The instructions which worked for me are:

1. Install the [Steam macOS client](https://store.steampowered.com/about/)
2. From the terminal, run Steam with the -console flag: /Applications/Steam.app/Contents/MacOS/steam_osx -console. This will give you a new `CONSOLE` tab at the top beside the `STORE`, `LIBRARY`, etc tabs.
3. Find [Fallout on steamdb.info](https://steamdb.info/app/38400/) and grab the app ID and the depot ID (for me the app ID was `38400` and the English language depot ID was `38409`--you can find more depots on the 'depots' tab on the left of the page)
4. In the `CONSOLE` tab of the Steam client, run `download_depot <appid> <depotid>` (e.g., `download_depot 38400 38409`). After a moment, the console should report that the download has begun and when it completes, it will print the download path.
5. Optionally move or copy the depot directory somewhere more accessible, like `~/Games/Fallout`
6. Download the Fallout CE installer (the macOS .dmg file) from the [Fallout CE releases page](https://github.com/alexbatalov/fallout1-ce/releases) if you haven't already. Running the installer should open a Finder window with just the Fallout Community Edition app; drag that into your depot directory (e.g., `~/Games/Fallout`).
7. Open your depot directory in a Finder window and double-click the Fallout Community Edition application to start the game.

**Comment by daltonmy** [comment]
I keep getting "could not open file" from innoextract. Anyone know a fix?

**Comment by tofagerl** [comment]
This looks like this to me: 
![image](https://github.com/alexbatalov/fallout1-ce/assets/781189/7c290f60-a138-47c8-ab43-bdfc87f44078)
And then I get this error: 
![image](https://github.com/alexbatalov/fallout1-ce/assets/781189/cc78a446-96aa-4a65-9794-65b3e23b8024)

**Comment by weberc2** [comment]
@tofagerl Did you download from Steam from the instructions I provided? When I downloaded from Steam I only had a MASTER.DAT, a CRITTER.DAT, and a fallout.cfg file...

**Comment by tofagerl** [comment]
@weberc2 No, I used Whisky and assumed I just did the same operation in a different way. 
I'll try it your way.

**Comment by tofagerl** [comment]
It seems to have more files, actually. Oh well. 
It works now - I just downloaded it with the console thing, and verified that it started up. 
I'm at work, so I can't really start playing yet :D

Thanks for your help!

**Comment by yr12345678** [comment]
Thanks @guy4261 I ran into the same issue. Got as far as actually extracting the offline installer instead of the GOG installer, but having to drag the .app file into the folder didn't click for me yet, haha. Works now :)
